test/integration/rules/multiple-label/multiple-label.html

```diff
@@ -0,0 +1,16 @@
+<!-- Fail -->
+<label for="fail1">Hi</label>
+<label for="fail1">Foo</label>
+<input type="text" id="fail1">
+
+<label for="fail2">label one</label>
+<label for="fail2">label two</label>
+<input type="checkbox" id="fail2">
+
+<label for="fail3" id="l1">label one</label>
+<label for="fail3">label two</label>
+<input type="checkbox" id="fail3" aria-labelledby="l1">
```

Comment:
Should add tests for:
- implicit + explicit label
- multiple implicit labels
- select elements